### PR TITLE
[zh] resync setup section [3]

### PR DESCRIPTION
**Pull request by howieyuen**
related: https://github.com/kubernetes/website/issues/27671

### CODE REVIEW

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 1]
```diff
@@ -1,264 +1,228 @@
...
-At {{< param "version" >}}, Kubernetes supports clusters with up to 5000 nodes. More specifically, we support configurations that meet *all* of the following criteria:
+A cluster is a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (physical
+or virtual machines) running Kubernetes agents, managed by the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
+Kubernetes {{< param "version" >}} supports clusters with up to 5000 nodes. More specifically,
+Kubernetes is designed to accommodate configurations that meet *all* of the following criteria:
 -->
-在 {{< param "version" >}} 版本中， Kubernetes 支持的最大节点数为 5000。更具体地说，我们支持满足以下*所有*条件的配置：
+集群是运行 Kubernetes 代理的、
+由{{< glossary_tooltip text="控制平面" term_id="control-plane" >}}管理的一组{{< glossary_tooltip text="节点" term_id="node" >}}（物理机或虚拟机）。
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:22]:
```suggestion
由{{< glossary_tooltip text="控制平面" term_id="control-plane" >}}管理的一组
{{< glossary_tooltip text="节点" term_id="node" >}}（物理机或虚拟机）。
```

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 2]
```diff
@@ -1,264 +1,228 @@
...
-{{< toc >}}
-
-<!--
-## Setup
+<!-- 
+You can scale your cluster by adding or removing nodes. The way you do this depends
+on how your cluster is deployed.
 -->
-## 设定
+你可以通过添加或删除节点来扩展集群。集群扩展的方式取决于集群的部署方式。
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:41]:
```suggestion
你可以通过添加或删除节点来扩缩集群。集群扩缩的方式取决于集群的部署方式。
```

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 3]
```diff
@@ -1,264 +1,228 @@
...
+你应该在每个故障区域至少应运行一个实例，以提供容错能力。 
+Kubernetes 节点不会自动将流量引向相同故障区域中的控制平面端点。 
+但是，你的云供应商可能有自己的机制来执行此操作。
 
-<!--
-On GCE/Google Kubernetes Engine, and AWS, `kube-up` automatically configures the proper VM size for your master depending on the number of nodes
-in your cluster. On other providers, you will need to configure it manually. For reference, the sizes we use on GCE are
--->
-在 GCE/Google Kubernetes Engine 和 AWS 上，`kube-up` 会根据节点数量自动为您集群中的 master 节点配置适当的虚拟机大小。在其它云供应商的平台上，您将需要手动配置它。作为参考，我们在 GCE 上使用的规格为：
+例如，使用受管空的负载均衡，你可以配置负载平衡发送源自故障区域 _A_ 中的 kubelet 和 Pod 的流量，
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:107]:
```suggestion
例如，使用托管的负载均衡器时，你可以配置负载均衡器发送源自故障区域 _A_ 中的 kubelet 和 Pod 的流量，
```

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 4]
```diff
@@ -1,264 +1,228 @@
...
-<!--
-On AWS, master node sizes are currently set at cluster startup time and do not change, even if you later scale your cluster up or down by manually removing or adding nodes or using a cluster autoscaler.
+* start and configure additional etcd instance
+* configure the {{< glossary_tooltip term_id="kube-apiserver" text="API server" >}} to use it for storing events
 -->
-在 AWS 上，主控节点的规格是在集群启动时设置的，并且，即使以后通过手动删除或添加节点的方式使集群缩容或扩容，主控节点的大小也不会更改。
-{{< /note >}}
+在创建集群时，你可以（使用自定义工具）：
+
+* 启动并配置其它 etcd 实例
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:131]:
```suggestion
* 启动并配置额外的 etcd 实例
```

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 5]
```diff
@@ -1,264 +1,228 @@
...
-Currently you have two choices: restart the cluster (`kube-down.sh` and then `kube-up.sh` again), or before
-running `kube-up.sh` set the environment variable `ALLOWED_NOTREADY_NODES` to whatever value you feel comfortable
-with. This will allow `kube-up.sh` to succeed with fewer than `NUM_NODES` coming up. Depending on the
-reason for the failure, those additional nodes may join later or the cluster may remain at a size of
-`NUM_NODES - ALLOWED_NOTREADY_NODES`.
--->
-出于各种原因（更多详细信息，请参见 [#18969](https://github.com/kubernetes/kubernetes/issues/18969)），
-在 `kube-up.sh` 中设置很大的 `NUM_NODES` 时，可能会由于少数节点无法正常启动而失败。
-此时，您有两个选择：重新启动集群（运行 `kube-down.sh`，然后再运行 `kube-up.sh`），或者在运行 `kube-up.sh` 之前将环境变量 `ALLOWED_NOTREADY_NODES` 设置为您认为合适的任何值。采取后者时，即使运行成功的节点数量少于 `NUM_NODES`，`kube-up.sh` 仍可以运行成功。根据失败的原因，这些节点可能会稍后加入集群，又或者群集的大小保持在 `NUM_NODES-ALLOWED_NOTREADY_NODES`。
+* 部分垂直扩展插件——集群有一个插件副本或服务于整个故障区域。
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:197]:
```suggestion
* 部分垂直扩展插件——总有一个插件副本服务于整个集群或服务于整个故障区域。
```

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 6]
```diff
@@ -1,264 +1,228 @@
...
+  对于这些附加组件，请在扩展集群时加大资源请求和资源限制。
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:198]:
```suggestion
  对于这些附加组件，请在扩大集群时加大资源请求和资源限制。
```

File: content/zh/docs/setup/best-practices/cluster-large.md [hunk 7]
```diff
@@ -1,264 +1,228 @@
...
+* 许多水平扩展插件——你可以通过运行更多的 Pod 来增加容量——但是在大规模集群下，
```
Comment (tengqm) [inline content/zh/docs/setup/best-practices/cluster-large.md:199]:
```suggestion
* 许多插件可以水平扩展 —— 你可以通过运行更多的 Pod 来增加容量——但是在大规模集群下，
```

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**